src/main/resources/static/js/app.js

```diff
@@ -0,0 +1,119 @@
+var app = angular.module("JournalApp", []);
+
+app.controller("CategoryController", function ($scope, $http) {
+
+    $scope.isAnySubscription = function() {
+        var result = false;
+        for(var i=0; i< $scope.subscriptions.length; i++) {
+            if($scope.subscriptions[i].active) {
+                return true;
+            }
+        }
+        return result;
+    }
+
+    $scope.subscribe = function (categoryId) {
+        $http.post('/rest/journals/subscribe/' + categoryId).success(function (data, status, headers, config) {
```

Comment:
If we can transpile this or IE support is not needed, we can just use string literal from ES2015 spec:

```javascript
`/rest/journals/subscribe/${categoryId}`
```